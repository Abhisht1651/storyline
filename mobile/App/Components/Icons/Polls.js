import React from 'react'
import { View, Text } from 'react-native'

// Styles
import styles from '../Styles/IconStyles/PollsStyles'
import PollSVG from '../../Images/Icons/icon-poll-dark.svg'

const Polls = () => {
  return (
    <View style={styles.cardBox}>
      <View style={styles.profileIconBox}>
        <PollSVG />
      </View>
      <View style={styles.TextBox}>
        <Text style={styles.TextStyles}>My Polls</Text>
      </View>
    </View>
  )
}

export default Polls
