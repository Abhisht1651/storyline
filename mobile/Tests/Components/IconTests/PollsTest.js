import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'

import Polls from '../../../App/Components/Icons/Polls'

test('Polls Icon component renders correctly', () => {
  const tree = renderer.create(<Polls />).toJSON()
  expect(tree).toMatchSnapshot()
})
