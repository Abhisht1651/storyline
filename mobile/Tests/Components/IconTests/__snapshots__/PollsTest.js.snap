// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Polls Icon component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "flexDirection": "row",
      "height": 20,
      "opacity": 1,
      "width": 110,
    }
  }
>
  <View
    style={
      Object {
        "height": 16,
        "width": 21,
      }
    }
  >
    <SvgMock />
  </View>
  <View
    style={
      Object {
        "height": 20,
        "marginLeft": 16,
        "width": 75,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#1F1F60",
          "fontFamily": "BananaGrotesk-Medium",
          "fontSize": 18,
          "letterSpacing": -0.12,
          "opacity": 1,
          "textAlign": "left",
        }
      }
    >
      My Polls
    </Text>
  </View>
</View>
`;
